js/form.js

```diff
@@ -0,0 +1,16 @@
+const form = document.getElementById('form');
+const email = document.getElementById('email_form');
+const error = document.querySelector('.error-email');
+const regex = /^[a-z0-9_-]+@[a-z0-9]+\.[a-z]+\.?[a-z]+/g;
+
+form.addEventListener('submit', (e) => {
+  if (regex.test(email.value)) {
+    error.textContent = '';
+    email.style.border = '1px solid #cfd8dc';
+  } else {
+    e.preventDefault();
+    email.style.border = '3px solid red';
```

Comment:
- [ ] Let's improve the contrast of the red text over the blu background. Currently, it has a value of 1 and it should get to at least 3, and better if you can take it over 4.5. Remember that this is helpful for people with sight problems, as daltonism. We could also make it fit a bit better in our design.

![image](https://user-images.githubusercontent.com/79114233/141154241-ab009756-0a21-49d8-a358-757201641f3f.png)

Reply:
Done